hack/tools/test_module.sh

```diff
@@ -0,0 +1,232 @@
+# cd ${PATH_TO_LOCAL_FYBRIK}
+# helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
+# helm install fybrik charts/fybrik --set global.tag=master --set global.imagePullPolicy=Always -n fybrik-system --wait
+
+
+# hello-world-module
+
+#datashim
+kubectl apply -f https://raw.githubusercontent.com/datashim-io/datashim/master/release-tools/manifests/dlf.yaml
+kubectl wait --for=condition=ready pods -l app.kubernetes.io/name=dlf -n dlf --timeout=500s
+
+
+sleep 10
+# kubectl apply -f hello-world-module.yaml -n fybrik-system
+kubectl apply -f https://github.com/fybrik/hello-world-module/releases/download/v$moduleVersion/hello-world-module.yaml -n fybrik-system
+
+
+# Notebook sample
+
+kubectl create namespace fybrik-notebook-sample
+kubectl config set-context --current --namespace=fybrik-notebook-sample
+
+#localstack
+helm repo add localstack-charts https://localstack.github.io/helm-charts
+helm install localstack localstack-charts/localstack --set startServices="s3" --set service.type=ClusterIP
+kubectl wait --for=condition=ready --all pod -n fybrik-notebook-sample --timeout=400s
+
+kubectl port-forward svc/localstack 4566:4566 &
+
+export ENDPOINT="http://127.0.0.1:4566"
+export BUCKET="demo"
+export OBJECT_KEY="PS_20174392719_1491204439457_log.csv"
+export FILEPATH="$WORKING_DIR/PS_20174392719_1491204439457_log.csv"
+aws configure set aws_access_key_id ${ACCESS_KEY} && aws configure set aws_secret_access_key ${SECRET_KEY} && aws --endpoint-url=${ENDPOINT} s3api create-bucket --bucket ${BUCKET} && aws --endpoint-url=${ENDPOINT} s3api put-object --bucket ${BUCKET} --key ${OBJECT_KEY} --body ${FILEPATH}
+
+
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: paysim-csv
+type: Opaque
+stringData:
+  access_key: "${ACCESS_KEY}"
+  secret_key: "${SECRET_KEY}"
+EOF
+
+
+cat << EOF | kubectl apply -f -
+apiVersion: katalog.fybrik.io/v1alpha1
+kind: Asset
+metadata:
+  name: paysim-csv
+spec:
+  secretRef: 
+    name: paysim-csv
+  details:
+    dataFormat: csv
+    connection:
+      name: s3
+      s3:
+        endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
+        bucket: "demo"
+        object_key: "PS_20174392719_1491204439457_log.csv"
+  metadata:
+    name: Synthetic Financial Datasets For Fraud Detection
+    geography: theshire 
+    tags:
+      finance: true
+    columns:
+      - name: nameOrig
+        tags:
+          PII: true
+      - name: oldbalanceOrg
+        tags:
+          PII: true
+      - name: newbalanceOrig
+        tags:
+          PII: true
+EOF
+
+
+#fybrikstorage
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bucket-creds
+  namespace: fybrik-system
+type: Opaque
+stringData:
+  access_key: "${ACCESS_KEY}"
+  accessKeyID: "${ACCESS_KEY}"
+  secret_key: "${SECRET_KEY}"
+  secretAccessKey: "${SECRET_KEY}"
+EOF
+
+cat << EOF | kubectl apply -f -
+apiVersion:   app.fybrik.io/v1alpha1
+kind:         FybrikStorageAccount
```

Comment:
I think `FybrikStorageAccount` in 0.5.0 version is different from 0.6.0 version so we will need two versions. Please look at branch `release/0.5.0` in hello-world module and fybrik 0.5.0 for details on the resources.